index.html

```diff
@@ -68,6 +68,8 @@
          * it */
         if (target) {
             var qr = new QRCode(target, {
+                width : 500,
```

Comment:
chromium or opera for example.

Check these screenshots from the same resolution: https://drive.google.com/file/d/0BxjHfqaf64A7S2RRRWNlUko3QTQ/view?usp=sharing
https://drive.google.com/file/d/0BxjHfqaf64A7UHQxYVhKcFVLMnc/view?usp=sharing